spec/rubocop/cop/rspec/stubbed_mock_spec.rb

```diff
@@ -126,12 +126,49 @@
     RUBY
   end
 
-  it 'tolerates passed arguments without parentheses' do
+  it 'flags even when passed arguments without parentheses' do
     expect_offense(<<~RUBY)
       expect(Foo)
       ^^^^^^^^^^^ Prefer `allow` over `expect` when configuring a response.
         .to receive(:new)
         .with(bar).and_return baz
     RUBY
   end
+
+  it 'flags `should_receive`', skip: 'Not implemented yet' do
+    expect_offense(<<~RUBY)
+      foo.should_receive(:bar).and_return(:baz)
+      ^^^^^^^^^^^^^^^^^^^ Prefer `allow` over `should_receive` when configuring a response.
+    RUBY
+  end
+
+  it 'flags `should_not_receive`',
+     skip: 'Not sure what to do with this kind of expectation' do
+    expect_offense(<<~RUBY)
+      foo.should_not_receive(:bar)
+      ^^^^^^^^^^^^^^^^^^^^^^ Prefer `allow(...).not_to receive` over `should_not_receive` when configuring a response.
+    RUBY
+  end
+
+  it 'flags `are_expected`',
+     skip: 'This passed before this commit. Handled like is_expected?' do
+    expect_offense(<<~RUBY)
+      are_expected.to receive(:bar).and_return(:baz)
```

Comment:
Is this from rspec-its? It’s soft-maintained, and is unlikely can be named an integral part of RSpec. I’d rather documented how to add it to the co figuration if the cop was configurable.

Reply:
Yes: https://github.com/search?q=org%3Arspec%20are_expected&type=code

My understanding is that it is included in the `Expectations.all` that the current Cop is using.